### PR TITLE
Add conversion notification

### PR DESCRIPTION
**Pull request by dinojs**
- Added notification if a project is successfully/unsuccessfully converted;
- After successfully converting the project, the notification will disappear within 2 seconds. 
image
image
image

### CODE REVIEW

File: src/views/WireframesPage.js [hunk 1]
```diff
@@ -86,31 +86,25 @@ export class WireframesPage extends Component {
...
-      }.bind(this),
-    });
+
+    api
+      .post(
+        `${BACKEND_ADDRESS}/api/v1/projects/${projectID}/build?authType=${authType}&accessToken=${accessToken}`,
+        { wireframeList: selected, userId: userID }
+      )
+      .then((res) => {
+        window.location.href = `${BACKEND_ADDRESS}/api/v1/projects/${projectID}/download`;
```
Comment (KashSingh1) [inline src/views/WireframesPage.js]: Shouldn't we change this to using history push instead? To follow the convention we use in other pages.

Comment (KashSingh1) [inline src/views/WireframesPage.js]: Shouldn't this be history push instead?

File: src/views/WireframesPage.js [hunk 2]
```diff
@@ -210,20 +226,42 @@ export class WireframesPage extends Component {
...
+            </div>
+          </div>
+        )}
+
+        {this.state.errorModal && (
+          <div className="d-flex justify-content-center align-items-center loader-background">
+            <div className="d-flex align-items-center flex-column loader-wrapper">
+              <h4>
+                Erorr building project!{" "}
+                <ExclamationCircleFill color="red" size={25} />
```
[the remark this answers is not in the document]
Comment (dinojs) [reply src/views/WireframesPage.js:251]: Do I have to do it or is it a suggestion? I'd rather keep it

[the remark this answers is not in the document]
Comment (KashSingh1) [reply src/views/WireframesPage.js:251]: Keep the icon. In the success modal, we have a green icon indicating success. To keep with this convention we also need a red icon to indicate failure. We can possibly discuss later about moving the place of the icon, but it should be there to follow the convention we have set.

**Comment by dinojs** [comment]
Vetter look and less cramped(?)
image
image

**Review by lukasotocerny — APPROVED** [review]
LGTM

**Pull request merged**